#goal definition
bool maintainPosture
---
#result definition
bool success
---
#feedback
float64 stabilityQuality